aikau/src/main/resources/alfresco/documentlibrary/_AlfDocumentListTopicMixin.js

```diff
@@ -178,6 +178,9 @@ define(["dojo/_base/declare",
        * @instance
        * @type {string} 
        * @default
+       * @event
+       * @property {string} [value] One of "selectAll", "selectNone", "selectInvert", "selectFolders" or "selectDocuments"
+       * @property {object[]} [selectedItems] An array of items that have been selected
```

Comment:
I thought we were going to use the @default tag to link people to the documentation that describes the event better (i.e. no default value and I think the property tags are redundant?)